.travis.yml

```diff
@@ -32,6 +32,7 @@ script:
   - cargo test --verbose --features=rand
   - cargo test --verbose --features="rand serde"
   - cargo test --verbose --features="rand serde recovery endomorphism"
+  - cargo test --lib --features=fuzztarget -- --test-threads=1 # 1 thread is required so that we can have 2 tests setting the `UNSAFE_CRYPTO_FUZZING` variable.
```

Comment:
Shouldn't this be `--verbose` instead of `--lib`?

Reply:
the `--lib` is to prevent the docs tests from running, because they don't pass.

I'll also add the --verbose